modules/_labs/badge/css/README.md

```diff
@@ -0,0 +1,68 @@
+# Canvas Kit CSS Badge
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+provides a quantity-based summary with dynamic values
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-css-badge
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-labs-css-badge/index.scss';
+```
+
+## Usage
+
+### Variants
+
+#### Default
+
+```html
+<span class="wdc-badge aria-live="polite" aria-label="1 unread notification">
+  1
+</span>
+```
+
+#### Inverse
+
+```html
+<span class="wdc-badge-inverse" aria-live="polite" aria-label="999+ unread notifications">
+  2
```

Comment:
Nit: does the aria label need to shorten as well? That's mainly for visual cleanliness
```suggestion
<span class="wdc-badge-inverse" aria-live="polite" aria-label="1000 unread notifications">
  1000
```